constants/appLinks.ts

```diff
@@ -57,6 +57,79 @@ const IBM_Q_EXPERIENCE: GeneralLink = {
   label: 'IBM Quantum Lab'
 }
 
+interface Prerequisite {
```

Comment:
This interface already exists on `PrerequisiteMaterialSection`, you don't need to create a new one